fastchat/serve/api_provider.py

```diff
@@ -156,6 +183,172 @@ def openai_api_stream_iter(
             yield data
 
 
+def upload_openai_file_to_gcs(file_id):
```

Comment:
rename to "upload_openai_image_file_to_gcs"?

Reply:
it could be csv, python, json etc so not necessary image.